content/en/docs/components/pipelines/v2/quickstart.md

```diff
@@ -1,16 +1,34 @@
 +++
 title = "Quickstart"
-description = "Getting started with Kubeflow Pipelines"
+description = "Get started with Kubeflow Pipelines"
 weight = 3
                     
 +++
+
+{{% stable-status %}}
+
 <!-- TODO: add UI screenshots for final pipeline -->
-In this tutorial we will deploy a KFP standalone deployment into an existing Kubernetes cluster, create and run a simple pipeline using the KFP SDK, view the pipeline on the KFP Dashboard, then create a more involved machine learning pipeline that uses additional KFP features.
+This tutorial helps you get started with KFP by doing the following:
+
+1. [Deploy a KFP standalone instance into an existing Kubernetes cluster.](#step-1-deploy-a-kfp-standalone-instance-into-an-existing-kubernetes-cluster)
```

Comment:
**Low-priority suggestion:** I'm not sure if the CMS supports this, but is there any way we could organize these into expandable hotspots that the user can click to view (similar to what we have on the following site)?

https://g3doc.corp.google.com/company/teams/cloud-devrel/docs/for-the-team/noogler-resources.md?cl=head#cloud-ix-noogler-checklist